crates/turborepo-lib/src/turbo_json/mod.rs

```diff
@@ -52,6 +55,12 @@ pub struct TurboJson {
 #[serde(rename_all = "camelCase")]
 // The raw deserialized turbo.json file.
 pub struct RawTurboJson {
+    #[serde(skip)]
+    // The raw text of the turbo.json file.
+    text: Option<Arc<str>>,
+    #[serde(skip)]
+    path: Option<Arc<str>>,
```

Comment:
Just to verify my understanding, the `text`/`path` inclusion in `TurboJson` and `RawTurboJson` is necessary as there can be parse errors at the validation step i.e. `RawTurboJson` -> `TurboJson` of afterwards when the `turbo.json` is being consumed by something like the task graph. Is this correct?

Reply:
Yeah there's some errors during validation that require these fields. Also since they're all `Arc`, it's pretty low cost to put them everywhere